test/test_modules.py

```diff
@@ -326,6 +327,7 @@ def inner_zero_grad(obj):
 
     @skipIfMps
     @modules(module_db)
+    @skipIfTorchInductor("to be fixed")
```

Comment:
are you working on these ? if not we should file issues

Reply:
I plan to file an umbrella issue for the top frequent failures. But I need to land this rng workaround to make it reproducible for people.